### ISSUE TITLE
Use JSON file within a reusable JSON file

### ISSUE THREAD

**Issue by felipeEngineer2020** [authorjapps/zerocode#453]
AC1:
===
Usage Documentation and example link: [In-progress...](#)

AC2:
===
As Automation Engineer I want to be able to reference a reusable JSON file from another reusable JSON file.

Example:
Part of test case:
```json
    {
      "id": "insert",
      "stepFile": "${JSON.FILE:reusable/step/**step1.json**}"
    }
```
And inside file **step1.json** I want to call a JSON file for the body

```json
"body": "${JSON.FILE:reusable/request/**body.json**}"
```
Contents of **body.json**
```json
{
  "id": 123,
  "name" : "test"
}
```
If this is already implemented please give me an example of the syntax to reference body.json. I tried the above but it did not work...Thanks!

**Comment by RDBreed** [comment]
Could also pick this up; know where to implement it (it isn't implemented as far as I could see). Just let me know, then I will create a PR for it!

**Comment by authorjapps** [comment]
Awaiting the release...
Available in `1.3.26` and higher versions.
